### PR TITLE
Fix codeowners file

### PR DESCRIPTION
**Pull request by jlf0dev**
Adds a `*` in front of the auth team for catch-all

### CODE REVIEW

**Review by JaredSnider-Bitwarden — APPROVED** [review]
👍

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 # The following owners will be the default owners for everything in the repo.
 # Unless a later match takes precedence
-@bitwarden/team-auth-dev
\ No newline at end of file
+* @bitwarden/team-auth-dev
```
Comment (withinfocus) [inline .github/CODEOWNERS:7]: ⚠️ I strongly prefer newlines at the ends of files.

**Review by withinfocus — DISMISSED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (jlf0dev) [reply .github/CODEOWNERS]: Fixed, had to turn off prettier for `codeowners` since it keeps trying to format it as markdown.

Comment (withinfocus) [reply .github/CODEOWNERS]: Yes, so annoying. Needs an exception in the config.

**Review by withinfocus — APPROVED** [review]

**Pull request merged**